lapis2/src/main/kotlin/org/genspectrum/lapis/OpenApiDocs.kt

```diff
@@ -90,6 +94,13 @@ fun buildOpenApiSchema(sequenceFilterFields: SequenceFilterFields, databaseConfi
                     requestSchemaWithFields(sequenceFilters, DETAILS_FIELDS_DESCRIPTION),
                 )
                 .addSchemas(
+                    NUCLEOTIDE_INSERTIONS_REQUEST_SCHEMA,
+                    requestSchemaForCommonSequencenFilters(sequenceFilters),
+                )
+                .addSchemas(
+                    AMINO_ACID_INSERTIONS_REQUEST_SCHEMA,
+                    requestSchemaForCommonSequencenFilters(sequenceFilters),
+                ).addSchemas(
```

Comment:
Maybe we could also use a single `INSERTION_REQUEST_SCHEMA` and reference to that in both endpoints? The schema seems to be identical anyway.